src/views/cron-task/update.php

```diff
@@ -5,14 +5,19 @@
 /* @var $this yii\web\View */
 /* @var $model gaxz\crontab\models\CronTask */
 
-$this->title = "Update ID:{$model->id} ({$model->route})";;
+$this->title = (!empty($model->name) ? 
+      "Update ID: {$model->id} - {$model->name}" 
+    : "Update ID: {$model->id} ({$model->route})");
 $this->params['breadcrumbs'][] = ['label' => 'Cron Tasks', 'url' => ['index']];
 $this->params['breadcrumbs'][] = ['label' => $model->id, 'url' => ['view', 'id' => $model->id]];
 $this->params['breadcrumbs'][] = 'Update';
 ?>
 <div class="cron-task-update">
 
     <h1><?= Html::encode($this->title) ?></h1>
+    <?php if (!empty($model->name)) { ?>
```

Comment:
```
<?php if (!empty($model->name)): ?>
<h4><?= Html::encode($model->name) ?></h4>
<?php else: ?>
<h4><?= Html::encode($model->route) ?></h4>
<?php endif; ?>
```
If\else statements should be written like that in template files.
And i guess there should be else, right?